addons/l10n_in/views/res_partner_views.xml

```diff
@@ -17,4 +14,15 @@
             </xpath>
         </field>
     </record>
+
+    <record id="l10n_in_view_partner_base_vat_form" model="ir.ui.view">
+        <field name="name">l10n.in.res.partner.base.vat.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base_vat.view_partner_base_vat_form"/>
```

Comment:
Why does this solve the problem by inheriting from another view?  (there seems to be no primary involved)

Reply:
because it replaces old one https://github.com/odoo/odoo/blob/17.0/addons/base_vat/views/res_partner_views.xml#L9

Reply:
@jco-odoo

Reply:
Because we are sure it will be applied after the change from base_vat that way...  

Would not it be better to put the l10n_in_gst_treatment also in here then?  @jva-odoo

Reply:
hmm, right. then no need for two views.
can we change the `inherit_id` in the already defined view?

Reply:
in my opinion, this issue should be solved more generally, as this is the second time there is a module that has an issue with that vat replacement. I will work on that :)
that being said, I noticed that `l10n_in_pan` is visible even if the active company is not Indian, I think it would be nice to fix that

Reply:
Hello @aliyatastemirova,
So is there any task that I can give reference to cancel the l10n_in function?

Reply:
> in my opinion, this issue should be solved more generally, as this is the second time there is a module that has an issue with that vat replacement. I will work on that :) that being said, I noticed that `l10n_in_pan` is visible even if the active company is not Indian, I think it would be nice to fix that

@aliyatastemirova it's possible from `SaaS-16.3`?

Reply:
> Hello @aliyatastemirova, So is there any task that I can give reference to cancel the l10n_in task?

there is no task, but I am reopening an old PR https://github.com/odoo/odoo/pull/138528

> it's possible from SaaS-16.3?

if you mean if it's possible to add the `invisible` attribute on `l10n_in_pan`, then yes :slightly_smiling_face: